### ISSUE TITLE
[12.0][14.0] modifica nome variabili

### ISSUE THREAD

**Issue by TheMule71** [OCA/l10n-italy#2510]
https://github.com/OCA/l10n-italy/blob/1fc7123eac6c0ec936a02f77431ea3c6ef8960ab/l10n_it_intrastat/models/account.py#L275-L284

Sebbene sia pressoché una questione estetica, il codice si basa sull'assunzione implicita che la valuta della company sia euro, sarebbe meglio prendere direttamente euro e usare quello per le conversioni, oppure cambiare nome alle variabili per essere un po' più agnostiche. Vd. anche

https://github.com/OCA/l10n-italy/pull/2228#discussion_r734567124

**Comment by sergiocorato** [comment]
Concordo che se si dà per scontato che siano Euro, sia meglio prendere direttamente la valuta ignorando l'impostazione dell'azienda.
Per correttezza sarebbe da far apparire un errore quando l'utente tenta di inserire una valuta diversa dall'Euro all'azienda se sono installati i moduli intrastat.
